tools/topology/topology2/production/tplg-targets-ace3.cmake

```diff
@@ -66,6 +66,19 @@ SDW_JACK_OUT_STREAM=Playback-SimpleJack,SDW_JACK_IN_STREAM=Capture-SimpleJack,\
 PREPROCESS_PLUGINS=nhlt,NHLT_BIN=nhlt-sof-ptl-rt722-2ch.bin,DMIC0_ENHANCED_CAPTURE=true,\
 EFX_DMIC0_TDFB_PARAMS=line2_generic_pm10deg,EFX_DMIC0_DRC_PARAMS=dmic_default"
 
+"cavs-sdw\;sof-ptl-rt722-l0-rt1320-l23\;PLATFORM=ptl,SDW_DMIC=1,NUM_SDW_AMP_LINKS=2,\
+SDW_AMP_FEEDBACK=false,SDW_SPK_STREAM=Playback-SmartAmp,SDW_DMIC_STREAM=Capture-SmartMic,\
+SDW_JACK_OUT_STREAM=Playback-SimpleJack,SDW_JACK_IN_STREAM=Capture-SimpleJack"
+
+"cavs-sdw\;sof-ptl-rt722-l0-rt1320-l23-4ch\;PLATFORM=ptl,SDW_DMIC=1,NUM_SDW_AMP_LINKS=2,NUM_DMICS=4,\
```

Comment:
Do we need `SDW_DMIC=1`? Will rt722 DMIC and PCH DMIC be coexist?

Reply:
Android/Chrome products reserve the codec dmic based on the hardware configuration. No coexistence -only one source can be selected per design.

Reply:
> Android/Chrome products reserve the codec dmic based on the hardware configuration. No coexistence -only one source can be selected per design.

Then, I think `SDW_DMIC=1` and `SDW_DMIC_STREAM=Capture-SmartMic` should be removed.

Reply:
Done.